src/main/webapp/js/playGame.js

```diff
@@ -357,22 +360,49 @@ function updateUserProgress() {
   fetch(request);
 }
 
+/**
+* Creates a button element that is used for the minimap.
+* @param text the text that should go in the button.
+* @return {Element} an HTML element.
```

Comment:
Can you be more descriptive here, what the HTML element? 
And line 364, whats the button?

Reply:
Clarified.